src/util/logging.cpp

```diff
@@ -176,12 +238,12 @@ void handleMessage(
 #ifdef MIXXX_DEBUG_ASSERTIONS_FATAL
         // re-send as fatal.
         // The "%s" is intentional. See -Werror=format-security.
-        qFatal("%s", array.constData());
+        qFatal("%s", input.toLocal8Bit().constData());
         return;
 #endif // MIXXX_DEBUG_ASSERTIONS_FATAL
     }
 
-    writeToLog(array, writeFlags);
+    writeToLog(type, context, input, WriteFlag::All);
```

Comment:
Why WriteFlag::All instead of writeFlags?

Reply:
because I'm stupid :D Fixed.